src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -698,4 +713,23 @@ public void receiveStartGame() {
         }
     }
 
+    public static void addPotions() {
+
+        BaseMod.addPotion(AttackPotionButClaw.class, Color.RED, Color.WHITE, Color.FIREBRICK, AttackPotionButClaw.POTION_ID);
+        BaseMod.addPotion(ClawPowerPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, ClawPowerPotion.POTION_ID);
+        BaseMod.addPotion(DrawClawsPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, DrawClawsPotion.POTION_ID);
+        BaseMod.addPotion(GenerateClawsPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, GenerateClawsPotion.POTION_ID);
```

Comment:
Currently this will make these potions show up for all characters. Need to pass a final argument of the character enum, `ThePackmaster.Enums.THE_PACKMASTER`.

And since these are specific to the claw pack, you should override `getPackPotions` in the `ClawPack` class to return a list of the potion IDs.

Reply:
(The functionality for getPackPotions doesn't fully work yet, but Pandemonium is working on it.)

Reply:
After looking at the potions themselves I see they use the Placeholder rarity... which prevents some issues but not all of them, since there are sources of potions that don't respect rarity. Specifically anything that uses `PotionHelper.getRandomPotion`, which returns a uniformly random potion -- so the events Knowing Skull, Lab, Woman in Blue, the relics Cauldron, Tiny House, and probably one or two other things.

I've done something like this for Corrupt the Spire and I prevented the potions from showing up by patching that method. We should do something similar here (in addition to marking these as Packmaster only potions, good to be safe). Here's the patch from Corrupt the Spire if you want to adapt it: https://github.com/modargo/corruptthespire/blob/f47f2a7b9ee5950e128d01960d519c6ed949fb7d/src/main/java/corruptthespire/patches/potions/FilterOutCorruptedPotionsPatch.java

Reply:
Also, if the idea is that tehse are a special rarity and don't normally show up (i.e. they only show up from Alclawmize), you can ignore what I said about getPackPotions. Sorry for any confusion.

Reply:
These are indeed completely exclusive to Alclawmize.  But it sounds like I still will need that patch.